src/pages/library/dataBrowser-utils.js

```diff
@@ -56,9 +47,16 @@ export const isDatarepoSnapshot = dataset => {
   return _.toLower(dataset['dcat:accessURL']).includes(datarepoSnapshotUrlFragment)
 }
 
-export const getConsortiumsFromDataset = dataset => _.map(dataCollection => dataCollection['dct:title'], dataset['TerraDCAT_ap:hasDataCollection'])
+export const getConsortiumsFromDataset = dataset => _.map('dct:title', dataset['TerraDCAT_ap:hasDataCollection'])
 
 
+export const getDataReleasePolicyFromDataset = dataset => _.has(dataset['TerraDCAT_ap:hasDataUsePermission'], datasetReleasePolicies) ?
```

Comment:
Now that we're adding more mapping functions, it seems like we'd want to organize these around the `dataset` object itself. Could this be a method in a `Dataset` class? I think it'd be OK to start with a type with only a few methods and add more over time as we migrate the mapping code from the normalize method.

Reply:
Yeah, that sounds reasonable. I think we have mostly done that via functions in the UI rather than classes, though. I have some ideas for how to do that, but honestly I might bring it to the UI mob on Thursday